### DISCUSSION TITLE
R Submissions Working Group - Minutes 2024-06-07

### DISCUSSION THREAD

**Discussion by giscus** [RConsortium/submissions-wg#115] [General]
# R Submissions Working Group - Minutes 2024-06-07

https://rconsortium.github.io/submissions-wg/minutes/2024-06-07/

**Comment by robertdevine** [comment]
Thank you!

**Comment by pzhang-cims** [comment]
Thanks for sharing the minutes that good to know there is a preparation for industry to share a 'good' R version and good R packages with their version for regulatory submissions. This can benefit all organizations for easier communication.
